carveracontroller/main.py

```diff
@@ -2616,10 +2765,11 @@ def setUIForModel(self, model, *args):
         if model != app.model:
             app.model = model.strip()
         if app.model == 'CA1':
-            self.tool_drop_down.set_dropdown.values = ['Probe','3D Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4', 'Tool: 5',
-                                                        'Tool: 6', 'Laser', 'Custom']
-            self.tool_drop_down.change_dropdown.values = ['Probe', '3D Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4',
-                                                            'Tool: 5', 'Tool: 6', 'Laser', 'Custom']
+            if app.is_community_firmware:
+                self.tool_drop_down.set_dropdown.values = ['Probe','3D Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4', 'Tool: 5',
+                                                            'Tool: 6', 'Laser', 'Custom']
+                self.tool_drop_down.change_dropdown.values = ['Probe', '3D Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4',
+                                                                'Tool: 5', 'Tool: 6', 'Laser', 'Custom']
             CNC.vars['rotation_base_width'] = 300
             CNC.vars['rotation_head_width'] = 56.5
         else:
```

Comment:
This is great, finally having a `is_community_firmware` attribute. Let's also add the `C1` scenario and leave the default dropdown with no 3d probe
```suggestion
     if app.model == 'CA1':
            if app.is_community_firmware:
                self.tool_drop_down.set_dropdown.values = ['Probe','3D Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4', 'Tool: 5',
                                                            'Tool: 6', 'Laser', 'Custom']
                self.tool_drop_down.change_dropdown.values = ['Probe', '3D Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4',
                                                                'Tool: 5', 'Tool: 6', 'Laser', 'Custom']
            CNC.vars['rotation_base_width'] = 300
            CNC.vars['rotation_head_width'] = 56.5
        elif app.model == 'C1:
            if app.is_community_firmware:
                self.tool_drop_down.set_dropdown.values = ['Probe','3D Probe', 'Tool: 1', 'Tool: 2', 'Tool: 3', 'Tool: 4', 'Tool: 5',
                                                            'Tool: 6', 'Laser', 'Custom']
```